scripts/zones/Alzadaal_Undersea_Ruins/IDs.lua

```diff
@@ -10,31 +10,31 @@ zones[xi.zone.ALZADAAL_UNDERSEA_RUINS] =
 {
     text =
     {
-        NOTHING_HAPPENS         = 119, -- Nothing happens...
+        NOTHING_HAPPENS         = 119,  -- Nothing happens...
         ITEM_CANNOT_BE_OBTAINED = 6383, -- You cannot obtain the <item>. Come back after sorting your inventory.
         ITEM_OBTAINED           = 6389, -- Obtained: <item>.
         GIL_OBTAINED            = 6390, -- Obtained <number> gil.
         KEYITEM_OBTAINED        = 6392, -- Obtained key item: <keyitem>.
+        NOTHING_OUT_OF_ORDINARY = 6403, -- There is nothing out of the ordinary here.
```

Comment:
thats a really weird shift, is it a duplicate of the same textID?

Reply:
Good point: my new tool matches based on the first exact match it finds in the comments and actual text.
If `There is nothing out of the ordinary here.` appears multiple times, it'll pick the lowest id it sees for it. Functionally, this shouldn't make any difference :shrug:

Reply:
awesome :)